Makefile

```diff
@@ -100,7 +97,7 @@ endif
 	find . -name \*~ -delete
 	find . -name \#\* -delete
 	rm -f bin/podman
-	make -C conmon clean
+	make -C clean
```

Comment:
This should be removed - we don't have a conmon folder anymore, so no need to invoke make again on that folder